cmd/kubelet/app/options/options.go

```diff
@@ -423,6 +432,7 @@ func (f *KubeletFlags) AddFlags(mainfs *pflag.FlagSet) {
 	fs.StringVar(&f.SeccompProfileRoot, "seccomp-profile-root", f.SeccompProfileRoot, "<Warning: Alpha feature> Directory path for seccomp profiles.")
 	fs.StringVar(&f.BootstrapCheckpointPath, "bootstrap-checkpoint-path", f.BootstrapCheckpointPath, "<Warning: Alpha feature> Path to the directory where the checkpoints are stored")
 	fs.Int32Var(&f.NodeStatusMaxImages, "node-status-max-images", f.NodeStatusMaxImages, "<Warning: Alpha feature> The maximum number of images to report in Node.Status.Images. If -1 is specified, no cap will be applied.")
+	fs.DurationVar(&f.VolumeOperationMaxBackOff.Duration, "volume-operation-max-backoff-time", f.VolumeOperationMaxBackOff.Duration, "The maximum backoff time of volume operation, default 2minutes+2seconds.")
```

Comment:
don't include the default in the description, the value of `f.VolumeOperationMaxBackOff.Duration` will be printed automatically as the default

Reply:
this should also move to where we register flags on the kubelet config struct, not on kubeletflags